### DISCUSSION TITLE
New Cave-Dwelling Creature Embedded in Terrain/Walls

### DISCUSSION THREAD

**Discussion by matthewpost** [FakeFishGames/Barotrauma#6518] [Ideas and suggestions]
It would be interesting to have a creature which is embedded in cave walls and preys on passers-by. Mechanically, it's fairly different from enemies that currently reside in caves. Imagine an enemy embedded in cave walls that sort of looks like a starfish fungus, or the front end of a nautilus or a jellyfish polyp. Alternatively, mature ballast flora embedded in cave walls that grabs swimmers with noodly tendrils. In general, it would be a static, opportunistic hunter that tentacles in unsuspecting visitors or silly crawlers.

My interest is in the mechanics, not the particular style. As long as it makes caves significantly less safe, and is equal or more unsettling to look at than ballast flora, it'll be fun. :)

Animals for inspiration:
- Starfish Fungus 
- Bobbit worm
- Jellyfish polyps
- Nautilus
- Pitcher Plant
- Trapdoor Spider
![image](https://user-images.githubusercontent.com/23487461/131266181-cecc8359-f4f4-4d98-8bab-219ce222051b.png)
![image](https://user-images.githubusercontent.com/23487461/131266185-674edf6e-833c-423f-a8c5-9bd305111fa1.png)
![image](https://user-images.githubusercontent.com/23487461/131266205-85302059-b91d-4a0f-bf0d-0545bf357b63.png)
![image](https://user-images.githubusercontent.com/23487461/131266210-45967c87-3b78-4913-8d1d-ec0605f11801.png)

**Comment by AshEdwardsDesign** [comment]
+1 

I really like the idea of some "ambush" style predator in caves.

How about something that can disguise itself to look like resources (minerals or harvestable plants). Perhaps with some way to distinguish it if you know what to look for. Could make for a nasty surprise for inexperienced players,  they go to mine a rock and suddenly it pounces at them 😁

**Reply by Hinny** [reply]
I like the image of a creature with a long tentacle that semi-slowly pulls a diver towards its mouth while the diver and its friends try to kill the creature before the diver has reached the mouth.
